pkg/utils/client/util.go

```diff
@@ -65,3 +65,18 @@ func MappingFor(restMapper meta.RESTMapper, resourceOrKindArg string) (*meta.RES
 
 	return mapping, nil
 }
+
+// MappingForResources is a wrapper of MappingFor.
+func MappingForResources(restMapper meta.RESTMapper, filters []string) ([]*meta.RESTMapping, []error) {
```

Comment:
Can we support resources' short name like `kubectl` for `filter`( or fucntion `MappingFor`)?( e.g. pod, po, pods all works in kubectl)
For better and easier to use as a CLI tool, I think this is worth to support. I can raise a issue and subsequent PR to track this enhancement if that makes sense to you.